lib/active_job_tenant_decorator.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module ActiveJobTenant
```

Comment:
We may need to move some of this out.  But a word of caution, Raven is specific to PALs and not generalizable to Hyku.